frontend-react/.env.development

```diff
@@ -18,3 +18,6 @@ REACT_APP_ENV=development
 INLINE_RUNTIME_CHUNK=false
 ; this will generate noise when packages have bad scss source maps
 GENERATE_SOURCEMAP=false
+
+REACT_APP_APPINSIGHTS_KEY=7ce107ea-c24a-49e2-a05f-2aaeb2e82f29
```

Comment:
I filed a DevOps ticket for a dedicated dev environment for testing analytics stuff locally, but it's admittedly pretty low priority 😬  Mind removing this for the time being?

Reply:
Yup, was not intended to be part of the PR.